experiments/configurations/pgvector-single-node.json

```diff
@@ -3,104 +3,88 @@
         "name": "pgvector-default",
         "engine": "pgvector",
         "connection_params": {},
-        "collection_params": {
-          "hnsw_config": { "m": 16, "ef_construct": 128 }
-        },
+        "collection_params": {},
         "search_params": [
-          { "parallel": 1, "search_params": { "hnsw_ef": 128 } }
+          { "parallel": 8, "search_params": { "hnsw_ef": 128 } }
```

Comment:
Updated the `parallel` options for `pgvector-default` to be the same as `qdrant-default` in `qdrant-single-node.json`

Reply:
It's not used for running the experiments. But okay :)